### DISCUSSION TITLE
App connecting to GitHub. How to prevent?

### DISCUSSION THREAD

**Discussion by udondan** [cawa-93/vite-electron-builder#458] [Q&A] [answered]
When I compile the app and run it, the app opens a connection to github.com. I cannot see what exactly it's trying to access. My firewall (LittleSnitch) is only showing the domain it wants to access:

![Bildschirmfoto 2021-09-27 um 15 54 11](https://user-images.githubusercontent.com/6443408/134922530-cb1df870-b7c9-4572-88cf-b29be6dc12e6.png)

Any idea what it's doing there and how to prevent it? I want my app fully self contained not accessing ANY remote resources.

**Comment by cawa-93** [comment, marked as answer]
It may be auto-updater check. Try to comment this https://github.com/cawa-93/vite-electron-builder/blob/d95b2aa2f1915d9e420e874e09060b0849358a46/packages/main/src/index.ts#L87-L93

**Reply by udondan** [reply]
Oh yes, of course, that makes perfect sense. Thanks for the quick reply!
